src/System.Net.Primitives/tests/FunctionalTests/IPAddressParsing.cs

```diff
@@ -10,81 +10,190 @@ public class IPAddressParsing
     {
         #region IPv4
 
-        [Fact]
-        public void ParseIPv4_Basic_Success()
-        {
-            Assert.Equal("192.168.0.1", IPAddress.Parse("192.168.0.1").ToString());
-        }
-
-        [Fact]
-        public void ParseIPv4_WithSubnet_Failure()
```

Comment:
Not sure I see the advantage of this change.

It's kind of hard to understand the new format and the reasons we're feeding Uri the `InlineData`. Maybe we should somehow preserve the test names.